plugins/headlamp-plugin/template/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "$${name}",
+  "version": "0.0.1",
+  "description": "Your Headlamp plugin",
+  "scripts": {
+    "start": "headlamp-plugin --watch",
+    "build": "headlamp-plugin"
+  },
+  "keywords": [
+    "headlamp",
+    "kinvolk",
```

Comment:
Please remove `kinvolk` as I am not sure it should be associated with all plugins people develop.

Reply:
This is done.